src/atom.rs

```diff
@@ -572,6 +620,6 @@ mod tests {
         let values = Arc::new(values) as Arc<dyn Array>;
 
         let result = encode_primitive_dyn(&values, 0).unwrap();
-        assert_eq!(result.borrow(), BytesText::new("2024-09-11T00:00:00.000Z"));
+        assert_eq!(result.borrow(), BytesText::new("2024-09-11T00:00:00.000"));
```

Comment:
Can you please add a similar test that checks that `Z` or `+XX:XX` offset is included when DataFusion's type includes a timezone?

Here is a [good example](https://github.com/kamu-data/kamu-cli/blob/d119d4267ec0787b2704fdcd87555398e5ebbaca/src/odf/data-utils/src/data/format/csv.rs#L377-L385) of initializing a timestamp array.